test/e2e/specs/appearance/theme__details-preview-activate.ts

```diff
@@ -20,9 +20,10 @@ import { Browser, Page } from 'playwright';
 declare const browser: Browser;
 
 describe( DataHelper.createSuiteTitle( 'Theme: Preview and Activate' ), () => {
-	const accountName = envVariables.TEST_ON_JETPACK
-		? 'jetpackUser'
-		: getTestAccountByFeature( envToFeatureKey( envVariables ) );
+	const accountName =
+		envVariables.JETPACK_TARGET === 'remote-site'
+			? 'jetpackUser'
+			: getTestAccountByFeature( envToFeatureKey( envVariables ) );
```

Comment:
**thought:non-blocking** - it occurred to me that we rely on this a lot, but do we actually have unit tests within `calypso-e2e` to verify this behavior?